### ISSUE TITLE
Generated samples are very bad

### ISSUE THREAD

**Issue by SunAnLan** [carpedm20/DCGAN-tensorflow#139]
/home/deepglint/samples/test_arange_1017.jpg
Its mostly like this why?

**Comment by carpedm20** [comment]
I think you uploaded wrong image? Which dataset that you've used and how the loss function goes and what command did you used?

**Comment by deartonym** [comment]
I have similar concern as well.
[http://imgur.com/a/ridfj](url)
I used an anime faces dataset, this image is what I get from samples at Epoch 120, in fact, the samples are pretty like each other after 20 epoch.

With the chainer implementation ([https://github.com/mattya/chainer-DCGAN](url)), 
we have samples at epoch 7 like:
[http://imgur.com/a/ridfj](url)

If you could give some idea about where this problem come from, it would be great.

Thanks.

**Comment by yangj1e** [comment]
I got similar issue. Any idea?

**Comment by yangj1e** [comment]
@deartonym your configuration is incorrect. It should be

    --input_width 96 --output_height 48

@carpedm20 For images smaller than the default `input_width`, in `center_crop` function in `utils.py`, the index `i` and `j` could be negative, which does not make sense.

**Comment by deartonym** [comment]
@yangj1e 
Thanks so much for noticing my mistake. It is a terribly careless one. I guess the input parameter names have been changed and I did not notice them.

In fact it should be 
`--input_height 96 --output_height 48`
since the input/output_width is determined by height if not defined.

Thanks again, and my issue is well solved.

**Comment by mxl1990** [comment]
I don't why input_height is 96, but output_height must be 48?

**Comment by kabrio** [comment]
I think the mistake was, that he used --image_size  instead of --input_height!

**Comment by kabrio** [comment]
might also try to set OPTION=0 
see #166
https://github.com/carpedm20/DCGAN-tensorflow/blob/master/main.py#L95

**Comment by npark** [comment]
Regardless of OPTION values, all samples look bad. Any one who solved?

**Comment by parkerzf** [comment]
Hey,
I tried different options. Only option 0 generate useful result. Also, if I change the range from -0,5 to 0.5 to -1 to 1 the result is much better. You can try it yourself.
